turtle_example/src/turtle_example/circler.py

```diff
@@ -0,0 +1,27 @@
+import rospy
+from geometry_msgs.msg import Twist
+from turtlesim.srv import TeleportAbsolute, TeleportAbsoluteRequest
+
+
+def main():
+    rospy.init_node("circler")
+    vel_pub = rospy.Publisher("/turtle1/cmd_vel", Twist, queue_size=1000)
+    client = rospy.ServiceProxy("/turtle1/teleport_absolute", TeleportAbsolute)
+
+    srv = TeleportAbsoluteRequest()
+    srv.x = 0
+    srv.y = 0.5
+    srv.theta = 0
+
+    if client.call(srv):
+        rospy.loginfo("Teleporting to 0, 0.5, 0")
```

Comment:
It turns out that teleport absolute starts relative to the bottom left corner, so the circler will actually look a bit better with some different numbers. In addition it's bad practice to copy the numbers in the log info instead of referencing them (I know you just kept this from my previous example 😅 )

```suggestion
    srv.x = 2
    srv.y = 3
    srv.theta = -45

    if client.call(srv):
        rospy.loginfo(f"Teleporting to {srv.x}, {srv.y}, {srv.theta}")
```